File: fix_text.py
import sys

with open('frontend/src/components/admin/PeriodsTab.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('{React.createElement("ion-icon", { name: "pencil-outline" })} S?a\n              hàng lo?t', '{React.createElement("ion-icon", { name: "musical-notes-outline" })} Ð?i nh?c\n              hàng lo?t')

content = content.replace('name: "pencil-outline",\n                  style: { marginRight: "4px" },\n                })}{" "}\n                S?a hàng lo?t', 'name: "musical-notes-outline",\n                  style: { marginRight: "4px" },\n                })}{" "}\n                Ð?i nh?c')

with open('frontend/src/components/admin/PeriodsTab.tsx', 'w', encoding='utf-8') as f:
    f.write(content)

File: fix_text.js
const fs = require('fs');
let text = fs.readFileSync('frontend/src/components/admin/PeriodsTab.tsx', 'utf8');

text = text.replace(/\{React\.createElement\("ion-icon", \{ name: "pencil-outline" \}\)\} S?a\s+hàng lo?t/g, '{React.createElement("ion-icon", { name: "musical-notes-outline" })} Ð?i nh?c\\n              hàng lo?t');

text = text.replace(/name: "pencil-outline",\s+style: \{ marginRight: "4px" \},\s+\}\)\}\{" "\}\s+S?a hàng lo?t/g, 'name: "musical-notes-outline",\\n                    style: { marginRight: "4px" },\\n                  })}{" "}\\n                  Ð?i nh?c hàng lo?t');

fs.writeFileSync('frontend/src/components/admin/PeriodsTab.tsx', text, 'utf8');
